packages/react-router-dom/modules/NavLink.js

```diff
@@ -50,12 +50,12 @@ NavLink.propTypes = {
   activeStyle: PropTypes.object,
   style: PropTypes.object,
   isActive: PropTypes.func,
-  ariaCurrent: PropTypes.oneOf(['page', 'step', 'location', 'true'])
+  'aria-current': PropTypes.oneOf(['page', 'step', 'location', 'date', 'time', 'true'])
```

Comment:
I wonder if we should be worrying about all the possible values and just doing a string type here.

Reply:
The current standard confix for `create-react-app` includes the `eslint-plugin-jsx-a11y` rule `jsx-a11y/aria-proptypes` activated by default. It already validates all `aria-*` values which, in this case, shows the following warning message for this rule at compile time:

```
The value for aria-current must be a single token from the following: page,step,location,date,time,true,false
```

So I agree that this is redundant.  Better to have such a dedicated library worry about these details.

Shall I remove it?